### ISSUE TITLE
Error thrown while performing the command - "tox -e check"

### ISSUE THREAD

**Issue by gutsytechster** [terminusdb/terminusdb-client-python#149]
**Describe the bug**
I am setting up the `terminusdb-client-python` repo and following this guide [here](https://github.com/terminusdb/terminusdb-client-python/blob/master/Contributing.md). However when I reach the step for `tox -e check`, it throws an error. I have performed all the steps as mentioned in that guide.

**To Reproduce**
Clone a fresh repo and perform the steps mentioned in the guide [here](https://github.com/terminusdb/terminusdb-client-python/blob/master/Contributing.md). When you reach the line `tox -e check` it throws an error. An additional step I took which is not mentioned in the guide, is to activate the virtualenv by running `pipenv shell`. Apart from this, all the steps are same.

**Expected behavior**
The check should be performed and should give the result.

**Error logs**
The error is thrown as
```
terminusdb-client-python on  master via 🐍 v3.8.2 (terminusdb-client-python-kX6_7a_V) took 1m48s 
❯ tox -e check
check installed: appdirs==1.4.4,attrs==20.2.0,autoflake==1.4,bandit==1.6.2,black==20.8b1,click==7.1.2,dataclasses==0.6,flake8==3.8.3,flake8-assertive==1.2.1,flake8-bandit==2.1.2,flake8-bugbear==20.1.4,flake8-builtins==1.5.3,flake8-comprehensions==3.2.3,flake8-polyfill==1.0.2,flake8-print==3.1.4,gitdb==4.0.5,GitPython==3.1.8,importlib-metadata==2.0.0,isort==5.5.4,libcst==0.3.11,mccabe==0.6.1,mypy-extensions==0.4.3,pathspec==0.8.0,pbr==5.5.0,pep8-naming==0.11.1,pybetter==0.3.5,pycodestyle==2.6.0,pyemojify==0.2.0,pyflakes==2.2.0,Pygments==2.7.1,pyupgrade==2.7.2,PyYAML==5.3.1,regex==2020.9.27,shed==0.2.4,six==1.15.0,smmap==3.0.4,stevedore==3.2.2,tokenize-rt==4.0.0,toml==0.10.1,typed-ast==1.4.1,typing-extensions==3.7.4.3,typing-inspect==0.6.0,zipp==3.2.0
check run-test-pre: PYTHONHASHSEED='459764776'
check run-test: commands[0] | shed
Traceback (most recent call last):
  File "/home/prashant/Documents/personal/OSS/terminusdb-client-python/.tox/check/bin/shed", line 5, in <module>
    from shed import cli
  File "/home/prashant/Documents/personal/OSS/terminusdb-client-python/.tox/check/lib/python3.8/site-packages/shed/__init__.py", line 46, in <module>
    from com2ann import com2ann
ModuleNotFoundError: No module named 'com2ann'
ERROR: InvocationError for command /home/prashant/Documents/personal/OSS/terminusdb-client-python/.tox/check/bin/shed (exited with code 1)
_____________________________________________________________________________________________________ summary _____________________________________________________________________________________________________
ERROR:   check: commands failed

```

**System information (please complete the following information):**
 - OS: Ubuntu 20.04
 -  terminus-client-python version: The editable version installed from `pip install -e .`

**Additional context**
None

**Comment by nyctophiliacme** [comment]
Getting the same issue. 

Error Logs:

```
(terminusdb-client-python) bash-3.2$ tox -e check
check create: /Users/tpransh/projects/open-source/terminusdb-client-python/.tox/check
check installdeps: --no-deps, -rrequirements-dev.txt
check installed: appdirs==1.4.4,attrs==20.2.0,autoflake==1.4,bandit==1.6.2,black==20.8b1,click==7.1.2,dataclasses==0.6,flake8==3.8.3,flake8-assertive==1.2.1,flake8-bandit==2.1.2,flake8-bugbear==20.1.4,flake8-builtins==1.5.3,flake8-comprehensions==3.2.3,flake8-polyfill==1.0.2,flake8-print==3.1.4,gitdb==4.0.5,GitPython==3.1.8,importlib-metadata==2.0.0,isort==5.5.4,libcst==0.3.11,mccabe==0.6.1,mypy-extensions==0.4.3,pathspec==0.8.0,pbr==5.5.0,pep8-naming==0.11.1,pybetter==0.3.5,pycodestyle==2.6.0,pyemojify==0.2.0,pyflakes==2.2.0,Pygments==2.7.1,pyupgrade==2.7.2,PyYAML==5.3.1,regex==2020.9.27,shed==0.2.4,six==1.15.0,smmap==3.0.4,stevedore==3.2.2,tokenize-rt==4.0.0,toml==0.10.1,typed-ast==1.4.1,typing-extensions==3.7.4.3,typing-inspect==0.6.0,zipp==3.2.0
check run-test-pre: PYTHONHASHSEED='1715063957'
check run-test: commands[0] | shed
Traceback (most recent call last):
  File "/Users/tpransh/projects/open-source/terminusdb-client-python/.tox/check/bin/shed", line 5, in <module>
    from shed import cli
  File "/Users/tpransh/projects/open-source/terminusdb-client-python/.tox/check/lib/python3.8/site-packages/shed/__init__.py", line 46, in <module>
    from com2ann import com2ann
ModuleNotFoundError: No module named 'com2ann'
ERROR: InvocationError for command /Users/tpransh/projects/open-source/terminusdb-client-python/.tox/check/bin/shed (exited with code 1)
____________________________________________________________________________________ summary _____________________________________________________________________________________
ERROR:   check: commands failed
```


OS: MacOS Mojave 10.14.6

**Comment by Cheukting** [comment]
I think I have seen that error before, it's some versioning problem. @gutsytechster and @nyctophiliacme can you let me know which Python version you are using?

**Comment by nyctophiliacme** [comment]
Hi @Cheukting . I am using Python version 3.8.2.

**Comment by Cheukting** [comment]
> Hi @Cheukting . I am using Python version 3.8.2.

ok, I will investigate, thanks.

**Comment by gutsytechster** [comment]
I am also using the same Python version i.e. 3.8.2

**Comment by cmaureir** [comment]
The problem, as @Cheukting mentioned, it's because[ the python_requires issues](https://github.com/ilevkivskyi/com2ann/commit/ce7583b5a98ee5b9dbeb8a8cef4239893e230291) with `com2ann` were fixed on `0.1.1`, and without that python_requires, the dependency for shed is `0.1.0`.
Adding `com2ann` as an independent package on the `requirements-dev.in` solves the issue, but I'm unfamiliar with tox, so I'm unsure if it's the recommended solution:

```
tox -e check
check create: /home/cmaureir/repos/terminusdb-client-python/.tox/check
check installdeps: --no-deps, -rrequirements-dev.txt
check installed: appdirs==1.4.4,attrs==20.2.0,autoflake==1.4,bandit==1.6.2,black==20.8b1,click==7.1.2,com2ann==0.1.1,dataclasses==0.6,flake8==3.8.3,flake8-assertive==1.2.1,flake8-bandit==2.1.2,flake8-bugbear==20.1.4,flake8-builtins==1.5.3,flake8-comprehensions==3.2.3,flake8-polyfill==1.0.2,flake8-print==3.1.4,gitdb==4.0.5,GitPython==3.1.8,importlib-metadata==2.0.0,isort==5.5.4,libcst==0.3.11,mccabe==0.6.1,mypy-extensions==0.4.3,pathspec==0.8.0,pbr==5.5.0,pep8-naming==0.11.1,pybetter==0.3.5,pycodestyle==2.6.0,pyemojify==0.2.0,pyflakes==2.2.0,Pygments==2.7.1,pyupgrade==2.7.2,PyYAML==5.3.1,regex==2020.9.27,shed==0.2.4,six==1.15.0,smmap==3.0.4,stevedore==3.2.2,tokenize-rt==4.0.0,toml==0.10.1,typed-ast==1.4.1,typing-extensions==3.7.4.3,typing-inspect==0.6.0,zipp==3.2.0
check run-test-pre: PYTHONHASHSEED='3555555568'
check run-test: commands[0] | shed
check run-test: commands[1] | flake8
________________________________________________________________ summary _________________________________________________________________
  check: commands succeeded
  congratulations :)

```

**Comment by Cheukting** [comment]
> The problem, as @Cheukting mentioned, it's because[ the python_requires issues](https://github.com/ilevkivskyi/com2ann/commit/ce7583b5a98ee5b9dbeb8a8cef4239893e230291) with `com2ann` were fixed on `0.1.1`, and without that python_requires, the dependency for shed is `0.1.0`.
> Adding `com2ann` as an independent package on the `requirements-dev.in` solves the issue, but I'm unfamiliar with tox, so I'm unsure if it's the recommended solution:
> 
> ```
> tox -e check
> check create: /home/cmaureir/repos/terminusdb-client-python/.tox/check
> check installdeps: --no-deps, -rrequirements-dev.txt
> check installed: appdirs==1.4.4,attrs==20.2.0,autoflake==1.4,bandit==1.6.2,black==20.8b1,click==7.1.2,com2ann==0.1.1,dataclasses==0.6,flake8==3.8.3,flake8-assertive==1.2.1,flake8-bandit==2.1.2,flake8-bugbear==20.1.4,flake8-builtins==1.5.3,flake8-comprehensions==3.2.3,flake8-polyfill==1.0.2,flake8-print==3.1.4,gitdb==4.0.5,GitPython==3.1.8,importlib-metadata==2.0.0,isort==5.5.4,libcst==0.3.11,mccabe==0.6.1,mypy-extensions==0.4.3,pathspec==0.8.0,pbr==5.5.0,pep8-naming==0.11.1,pybetter==0.3.5,pycodestyle==2.6.0,pyemojify==0.2.0,pyflakes==2.2.0,Pygments==2.7.1,pyupgrade==2.7.2,PyYAML==5.3.1,regex==2020.9.27,shed==0.2.4,six==1.15.0,smmap==3.0.4,stevedore==3.2.2,tokenize-rt==4.0.0,toml==0.10.1,typed-ast==1.4.1,typing-extensions==3.7.4.3,typing-inspect==0.6.0,zipp==3.2.0
> check run-test-pre: PYTHONHASHSEED='3555555568'
> check run-test: commands[0] | shed
> check run-test: commands[1] | flake8
> ________________________________________________________________ summary _________________________________________________________________
>   check: commands succeeded
>   congratulations :)
> ```

Thanks @cmaureir

**Comment by Cheukting** [comment]
I have checked and the latest version of shed now uses com2ann 0.1.1 (https://github.com/Zac-HD/shed/blob/master/setup.py) so I think it's ok as long as we just used the latest shed.
